### PR TITLE
Add case for evmctl ima_sign options testing

### PR DESCRIPTION
**Pull request by wnereiz**
Add to testsuite 'ima_appraisal'

- Related ticket: https://progress.opensuse.org/issues/50333
- Manual test case in Testopia: https://bugzilla.suse.com/tr_show_case.cgi?case_id=1699733
- Verification run: http://10.67.17.9/tests/880

### CODE REVIEW

File: tests/security/ima/evmctl_ima_sign.pm [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, see <http://www.gnu.org/licenses/>.
+#
+# Summary: Test evmctl ima_sign options
+# Note: This case should come after 'ima_apprasial_digital_signatures'
+# Maintainer: wnereiz <wnereiz@member.fsf.org>
+# Tags: poo#
```
Comment (lilyeyes) [inline tests/security/ima/evmctl_ima_sign.pm]: The poo number is missing.

File: tests/security/ima/evmctl_ima_sign.pm [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+        validate_script_output "getfattr -m . -d $g", sub {
+            # Base64 armored security.ima content (358 chars), we do not match
+            # the last three ones here for simplicity
+            m/security\.ima=[0-9a-zA-Z+\/]{355}/;
+        };
+    }
+
+    # Test -f (--sigfile) option
+    assert_script_run "evmctl -a sha256 ima_sign -p$mok_pass -k $mok_priv -f $sample_file1";
+    assert_script_run("test -e $sample_file1.sig",                    fail_message => 'signature file (.sig) has not been created');
```
Comment (lilyeyes) [inline tests/security/ima/evmctl_ima_sign.pm]: s/signature/Signature/

Comment (wnereiz) [reply tests/security/ima/evmctl_ima_sign.pm]: Thank you for pointing it out.

File: tests/security/ima/evmctl_ima_sign.pm (hunk 1 above)
Comment (wnereiz) [reply tests/security/ima/evmctl_ima_sign.pm]: Added,thanks

**Comment by lemon-suse** [comment]
LGTM.

**Pull request merged**